RedHat-UX/red-hat-design-system
# docs(tile): update best practices

## What I did

1. Updated tile's "Best practices" content to use a do/don't format


## Testing Instructions

1. Check tile's ["Best practices" section](https://deploy-preview-2033--red-hat-design-system.netlify.app/elements/tile/guidelines/#best-practices) in the DP.

## Notes to Reviewers
